src/borg/testsuite/repository.py

```diff
@@ -91,10 +95,9 @@ def repo_dump(repository, label=None):
     print()
 
 
-@pytest.mark.parametrize("fixture", ["repository", "remote_repository"])
-def test_basic_operations(fixture, request):
-    repo = request.getfixturevalue(fixture)
-    with repo as repository:
+@pytest.mark.parametrize("fixture", [pytest.lazy_fixture("repository"), pytest.lazy_fixture("remote_repository")])
```

Comment:
This works, however when discussing with @m3nu we were skeptical about including this as it seems like a small project that could be compromised: https://github.com/TvoroG/pytest-lazy-fixture

Reply:
Or abandoned and unmaintained, which sadly happens too often for useful packages.

Reply:
That lazy-fixture project doesn't look like regularly maintained. Last release 3.5y old.